server/webhook/jobs.go

```diff
@@ -0,0 +1,66 @@
+package webhook
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/xanzy/go-gitlab"
+)
+
+func (w *webhook) HandleJobs(ctx context.Context, event *gitlab.JobEvent) ([]*HandleWebhook, error) {
+	handlers, err := w.handleChannelJob(ctx, event)
+	if err != nil {
+		return nil, err
+	}
+	return cleanWebhookHandlers(handlers), nil
+}
+
+func (w *webhook) handleChannelJob(ctx context.Context, event *gitlab.JobEvent) ([]*HandleWebhook, error) {
+	senderGitlabUsername := event.User.Name
+	repo := event.Repository
+	res := []*HandleWebhook{}
+	message := ""
+	message = fmt.Sprintf("### Pipeline Job Stage: **%s**\n", event.BuildStage)
+
+	switch event.BuildStatus {
+	case statusRunning:
+		message += fmt.Sprintf(":rocket: **Status**: %s\n", event.BuildStatus)
+	case "pending":
+		message = fmt.Sprintf("### Pipeline Job Stage: **%s**\n", event.BuildStage)
+		message += fmt.Sprintf(":clock1: **Status**: %s\n", event.BuildStatus)
+	case statusSuccess:
+		message = fmt.Sprintf("### Pipeline Job Stage: **%s**\n", event.BuildStage)
+		message += fmt.Sprintf(":large_green_circle: **Status**: %s\n", event.BuildStatus)
+	case statusFailed:
+		message = fmt.Sprintf("### Pipeline Job Stage: **%s**\n", event.BuildStage)
+		message += fmt.Sprintf(":red_circle: **Status**: %s\n", event.BuildStatus)
+		message += fmt.Sprintf("**Reason Failed**: %s\n", event.BuildFailureReason)
+	default:
+		return res, nil
+	}
+	message += fmt.Sprintf("**Triggered By**: %s\n", senderGitlabUsername)
+	message += fmt.Sprintf("**Visit job [here](%s)**: \n", w.gitlabRetreiver.GetJobURL(repo.PathWithNamespace, event.BuildID))
```

Comment:
Not sure if the colon means anything here

```suggestion
	message += fmt.Sprintf("**Visit job [here](%s)**\n", w.gitlabRetreiver.GetJobURL(repo.PathWithNamespace, event.BuildID))
```

Reply:
I think it's just more readable.

Reply:
fixed